cmd/podman/common.go

```diff
@@ -132,7 +132,7 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 		"Drop capabilities from the container",
 	)
 	createFlags.String(
-		"cgroupns", "host",
+		"cgroupns", "",
```

Comment:
Can't we dynamically set the default like we do for a few other things? A bit less ugly than special-casing the empty string

Reply:
I was thinking about that, but one potential issue would be, if we default the CLI based on when the container was created, and the user switched cgroups, it could cause issues with existing containers.

Reply:
I'd prefer we don't add this logic to the `cmd/` package if possible.

Also, we need to be able to catch errors in the cgroups v2 detection.  If we move it here, we would need to either ignore the error or panic.